### PR TITLE
updated caption

### PR DESCRIPTION
**Pull request by ChauLam608**
test

### CODE REVIEW

**Comment by ChauLam608** [comment]
test

**Comment by khamvinh2004** [comment]
Awesome!!!

File: _slides/03-ChauLam608.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hello Chau Lam
```
Comment (khamvinh2004) [inline _slides/03-ChauLam608.md:6]: Need Some Comments

Comment (khamvinh2004) [inline _slides/03-ChauLam608.md:6]: Looks good!

**Review by khamvinh2004 — APPROVED** [review]

**Pull request merged**